pageserver/src/tenant/storage_layer/layer.rs

```diff
@@ -1496,6 +1510,26 @@ impl Default for LayerImplMetrics {
         )
         .unwrap();
 
+        let redownload_after = {
+            let minute = 60.0;
+            let hour = 60.0 * minute;
+            metrics::register_histogram!(
+                "pageserver_layer_redownloaded_after",
+                "Time between evicting and re-downloading.",
+                vec![
+                    10.0,
+                    30.0,
+                    minute,
+                    5.0 * minute,
+                    15.0 * minute,
+                    30.0 * minute,
+                    hour,
+                    12.0 * hour,
```

Comment:
@jcsp Notes mentioned you saying 20-60 minutes, but this PR adds buckets of 15min, 30min simply because it's nicer to split since we know nothing of the redownload time distribution. Should the buckets instead be every ten minutes?

Reply:
Well, I'm merging anyways because we can always change later.